common/history/characters/0_dvg_atlantia.txt

```diff
@@ -0,0 +1,34 @@
+CHARACTERS = {

```

Comment:
Some errors related to characters:
[07:24:31][jomini_effect.cpp:129]: PostValidate of effect 'add_radicals' returned false at  file: events/dvg_lotharingia.txt line: 254
[07:24:31][character_templates_utils.cpp:29]: Invalid Character trait provided orator in  file: events/dvg_lotharingia.txt line: 318
[07:24:31][character_templates_utils.cpp:29]: Invalid Interest Group ig_intelligentista in  file: events/dvg_lotharingia.txt line: 318
[07:24:31][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: events/dvg_lotharingia.txt line: 318
[07:24:31][character_templates_utils.cpp:29]: Invalid Ideology traditonalist in  file: common/history/characters/0_dvg_atlantia.txt line: 3
[07:24:31][character_templates_utils.cpp:29]: Can only do leader Ideologies in  file: common/history/characters/0_dvg_atlantia.txt line: 3
[07:24:31][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_atlantia.txt line: 3
[07:24:31][character_templates_utils.cpp:29]: Invalid Ideology moderate in  file: common/history/characters/0_dvg_atlantia.txt line: 18
[07:24:31][character_templates_utils.cpp:29]: Can only do leader Ideologies in  file: common/history/characters/0_dvg_atlantia.txt line: 18
[07:24:31][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_atlantia.txt line: 18